### PR TITLE
Adding recyler view

### LINKED ISSUE #18: Add Recycler View in home screen
1. Make a array list of approx 10 items.
2. Implement arraylist in home screen.
3. Load the arraylist with dummy array list you made.
PS : we'll use this array list (with modifications) to display our to-do list !!

### PR DESCRIPTION
**Pull request by BinduVerma123**
Closes #18
 -
 
### What changes are introduced to solve this?
 -
      
### Does this introduce a breaking change?
 - [ ] Yes
 - [x] No

### Pull Request Type
What kind of change does this Pull Request introduce?
 - [ ]  Bugfix
 - [ ]  Feature
 - [x]  Code style update
 - [ ]  Refactoring
 - [ ]  Documentation content changes
 - [ ]  Other… Please describe:

### How to Test the Code?
 - Describe:

### Please check if the PR fulfills these requirements
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
	
Other Information :
 -

### CODE REVIEW

**Comment by DreamyPhobic** [comment]
Add issue no after "Closes #" in pull request discription.

**Comment by DreamyPhobic** [comment]
After doing all changes as suggested, squash all commits into one commit.

**Comment by BinduVerma123** [comment]
@DreamyPhobic changes made

File: app/src/main/java/com/example/shivansh/to_dolist/MainActivity.java [hunk 1]
```diff
@@ -8,13 +8,18 @@
...
 import android.view.MenuItem;
+import android.support.v7.widget.LinearLayoutManager;
 
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        RecyclerView todoList = findViewById(R.id.todoList);
```
Comment (DreamyPhobic) [inline app/src/main/java/com/example/shivansh/to_dolist/MainActivity.java]: RecyclerView class is still not imported.

File: app/src/main/java/com/example/shivansh/to_dolist/ToDoTaskAdapter.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    public ToDoTaskAdapter(String[] data)
+    {
+        this.data = data;
+    }
+
+    @NonNull
+    @Override
+    public ToDoTaskViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int i) {
+        LayoutInflater inflater = LayoutInflater.from(parent.getContext());
+        View view = inflater.inflate(R.id.item_todo, parent, false);
```
Comment (DreamyPhobic) [inline app/src/main/java/com/example/shivansh/to_dolist/ToDoTaskAdapter.java]: At the place of R.id.item_todo, there will be R.layout.item_todo.

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -5,6 +5,8 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".MainActivity">
+    
+
```
Comment (DreamyPhobic) [inline app/src/main/res/layout/activity_main.xml]: Remove these extra spaces.

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -19,6 +21,15 @@
...
     </android.support.design.widget.AppBarLayout>
+    
+    <android.support.v7.widget.RecyclerView
+            xmlns:android="http://schemas.android.com/apk/res/android"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:id="@+id/todoList">
+
+
+        </android.support.v7.widget.RecyclerView>
```
Comment (DreamyPhobic) [inline app/src/main/res/layout/activity_main.xml]: Still it is overlaped by toolbar, add margin from top to it.

**Comment by DreamyPhobic** [comment]
It seems that you have directly made these changes without building it in android studio again... From next time, please first build and run the app in your device before submitting any PR.

**Comment by BinduVerma123** [comment]
@DreamyPhobic changes made

File: app/src/main/java/com/example/shivansh/to_dolist/MainActivity.java [hunk 5]
```diff
@@ -18,7 +19,7 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
         RecyclerView todoList = findViewById(R.id.todoList);
         todoList.setLayoutManager(new LinearLayoutManager(this));
-        String[] taskname = {"Task1", "Task2", "Task3", "Task4", "Task5"};
+        String[] taskname = {"Task1", "Task2", "Task3", "Task4", "Task5","Task6","Task7","Task8","Task9","Task10};
```
Comment (DreamyPhobic) [inline app/src/main/java/com/example/shivansh/to_dolist/MainActivity.java]: Do you think it is correct ??

Comment (DreamyPhobic) [reply app/src/main/java/com/example/shivansh/to_dolist/MainActivity.java]: RecyclerView is still overlaped by toolbar, add about 60dp top margin.

**Comment by BinduVerma123** [comment]
@DreamyPhobic changes made

**Pull request merged**